src/client-scopes/details/ScopeForm.tsx

```diff
@@ -49,9 +49,15 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
   const [openType, setOpenType] = useState(false);
   const { id } = useParams<{ id: string }>();
 
+  const displayOnConsentScreenEnabled =
+    clientScope["attributes"]?.["display.on.consent.screen"];
   const displayOnConsentScreen = useWatch({
     control,
     name: "attributes.display.on.consent.screen",
+    defaultValue:
+      displayOnConsentScreenEnabled == undefined
+        ? "true"
+        : displayOnConsentScreenEnabled,
```

Comment:
I think you can shorten your code here by getting rid of the intermediate variable. If you cast the attribute here directly to a boolean and then back to a string it will automatically turn undefined values into `false`.

```suggestion
    defaultValue: Boolean(clientScope.attributes?.["display.on.consent.screen"]).toString(),
```

Reply:
@jonkoops Sure, looks better. Thanks!